nifi-nar-bundles/nifi-accumulo-bundle/nifi-accumulo-processors/src/main/java/org/apache/nifi/accumulo/processors/ScanAccumulo.java

```diff
@@ -143,6 +150,16 @@
             .addValidator(Validator.VALID)
             .build();
 
+    static final PropertyDescriptor VALUE_INCLUDED_IN_RESULT = new PropertyDescriptor.Builder()
+            .displayName("Value Included in Result")
+            .name("accumulo-value-inclusive")
+            .description("Beside keys and their values, accumulo value field will also be included in the result as UTF-8 Encoded String.")
+            .required(false)
+            .defaultValue("True")
+            .allowableValues("True", "False")
```

Comment:
```suggestion
            .allowableValues("true", "false")
```

Reply:
@pvillard31 thanks for your comment however within this processor all boolean type properties start with capital letters, so I followed the pattern to stay consistent.